packages/components/src/Drawer/Drawer.component.js

```diff
@@ -93,8 +93,8 @@ function DrawerTitle({ title, children, onCancelAction }) {
 		return null;
 	}
 	return (
-		<div>
-			<div className={theme['tc-drawer-header']}>
+		<div className={theme['tc-drawer-header']}>
+			<div className={theme['tc-drawer-header-title']}>
```

Comment:
If we can have ellipsis, I suggest to deal with `title` attribute to display full content if needed.